.pipelines/e2e.yml

```diff
@@ -23,10 +23,16 @@ jobs:
       - script: |
           export SECRET_SA_ACCOUNT_NAME=$(SECRET_SA_ACCOUNT_NAME)
           make secrets
+          . secrets/env
+          az account set -s $AZURE_SUBSCRIPTION_ID
```

Comment:
We should move this into the e2e helper file. This step should only do `Downloading certificates and secrets from storage account` 
What thinking to do something like this : https://github.com/Azure/ARO-RP/pull/238/files#diff-ae0fbc8a6199a6bfda9913b986d5bf6fR157

WDYT ?